### PR TITLE
Fixed issue 181

### PR DESCRIPTION
**Pull request by glembus**
Fixed issue: [181](https://github.com/graphaware/neo4j-php-ogm/issues/181).
Added missed methods that implemented in EntityManager to EntityManagerInterface.

### CODE REVIEW

File: src/EntityManager.php [hunk 1]
```diff
@@ -87,6 +87,16 @@ class EntityManager implements EntityManagerInterface
      */
     protected $entityPersisters = [];
 
+    /**
+     * EntityManager constructor.
+     *
+     * @param ClientInterface                          $databaseDriver
+     * @param null                                     $cacheDirectory
```
Comment (ikwattro) [inline src/EntityManager.php]: should be string

Comment (glembus) [reply src/EntityManager.php]: fixed.

**Comment by glembus** [comment]
How I can run check for cla-bot. I have submitted signed License on email about 2 weeks ago.

**Comment by ikwattro** [comment]
The CLA is ok. It doesn't update this PR tho. Merging.

**Comment by ikwattro** [comment]
Thank you !

**Pull request merged**